signac/contrib/job.py

```diff
@@ -224,7 +224,7 @@ class Job:
     Application developers should not directly instantiate this class, but
     use :meth:`~signac.Project.open_job` instead.
 
-    Jobs can be opened by ``statepoint`` or ``_id``. If both values are
+    Jobs can be opened by ``statepoint`` or ``id_``. If both values are
```

Comment:
There should probably be a changelog entry to notify of this change.

Reply:
@lyrivera How would you like the changelog entry to be implemented?

Reply:
@ParisaTork a comment on `changelog.txt` notifying users that the keyword `_id` of the `Job` constructor has been renamed to `id_` should do

Reply:
Thank you so much - this has been sorted